source/features/link-to-changelog-file.tsx

```diff
@@ -63,9 +58,7 @@ async function init(): Promise<void | false> {
 	].join(',');
 
 	const navbar = (await elementReady(releasesOrTagsNavbarSelector, {waitForChildren: false}))!;
```

Comment:
```suggestion
	const navbar = await elementReady(releasesOrTagsNavbarSelector);
```

Since now it's being added inside, children matter.